test/test_force.js

```diff
@@ -290,7 +290,7 @@ function createCompileApp(tmpDir, os, actualAppType, templateRepoUri, pluginRepo
     var isHybridRemote = actualAppType === APP_TYPE.hybrid_remote;
     var templateName = getTemplateNameFromUri(templateRepoUri);
     var target = actualAppType + ' app for ' + os + (templateRepoUri ? ' based on template ' + templateName : '');
-    var appName = 'App_' + templateName.replace('#', '_') + '_' + os;
+    var appName = 'App_' + (templateRepoUri ? templateName.replace('#', '_') : actualAppType) + '_' + os;
```

Comment:
Same as below, calling functions on null if user chooses to supply app type instead of template.